apps/vaporgui/VSection.h

```diff
@@ -19,11 +19,13 @@ class VSection : public QTabWidget {
     Q_OBJECT
 
     class SettingsMenuButton;
+    class ExpandSectionButton;
 
 public:
     VSection(const std::string &title);
     QVBoxLayout *layout() const;
     void         setMenu(QMenu *menu);
+    void         setExpandedSection();
```

Comment:
Method is called `set...` but nothing is being set.

Reply:
QTabWidget::setCornerWidget() is being called though(?).

Reply:
For example, the method above it called "setMenu" sets the menu to the passed parameter. Maybe `enableExpandedSection` or `setExpandable(bool)` would be more appropriate?